manifests/bosh-manifest/reference-bosh-manifest.yml

```diff
@@ -191,5 +191,5 @@ resource_pools:
   name: vms
   network: private
   stemcell:
-    sha1: 89a2210b8caf3884855d7db6d48b8863202c7783
-    url: https://bosh.io/d/stemcells/bosh-aws-xen-hvm-ubuntu-trusty-go_agent?v=3197
+    sha1: 84c51fed6342d5eb7cd59728c7d691c75b6c1de8
+    url: https://bosh.io/d/stemcells/bosh-aws-xen-hvm-ubuntu-trusty-go_agent?v=3215
```

Comment:
These changes should be squashed into the commits that made the corresponding changes to the actual manifest.

Having said that, I'm not sure I see much value in the test that compares with the reference manifest. It's the sort of thing that's useful when initially refactoring manifests and sprucing them up, but I'm not sure it has value once that task is complete. We could consider removing this test entirely.